apps/frontend/src/libs/constants/constants.ts

```diff
@@ -1 +1,3 @@
+export { DEBOUNCE_TIMEOUT } from "./debounce-timeout.js";
 export { SIDEBAR_ITEMS } from "./navigation-items.constant.js";
+export { EMPTY_ARRAY_LENGTH } from "@git-fit/shared";
```

Comment:
Maybe `EMPTY_LENGTH` sounds more general? I used this name in my PR. However, it was not approved yet, so I'm not sure, how it should be called. Check the [comment](https://github.com/BinaryStudioAcademy/bsa-2024-gitfit/pull/115#discussion_r1738431638)

Reply:
agreed